lib/survey_monkey/question/checkboxes.rb

```diff
@@ -0,0 +1,32 @@
+module SurveyMonkey
+  module Question
+    class Checkboxes < Base
+      def initialize(id, answers:, **_args)
```

Comment:
as with other incarnations of this I think it is clearer to either pass all `options` and then use or set var from `options[:answers]` or  call with `answers: options[:answers]`, and remove the unused `**_args` from the definition